### DISCUSSION TITLE
Show to-be-created items in other relational fields to the same collection

### DISCUSSION THREAD

**Discussion by magomimmo** [directus/directus#3516] [Feature Requests]
Suppose to have a city collection with  a name property only (just for simplification). 
Then, suppose to have a person collection with a name and two cities properties: one to record her/his birthplace and the other to record her/his residence place. 

When you create a new entry for a person, if she/he has the residence in the same city of her/his birthplace and that city is not already available, but it's being created while creating the person, you don't find the city created as birthplace while creating the person's residence city and if you create the city again, you end up with two duplicated city.

---

### Project details
Directus Version: 9.0.0-rc.26
Environment: production
OS: Linux 5.4.0-58-generic
Node: 14.15.3

**Comment by benhaynes** [comment]
If you create it in one field/interface, that _staged_ value should also be available to other fields on the page.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
